File: BSharp/Controllers/DTO/Tenant.cs
namespace BSharp.Controllers.DTO
{

    public class TenantForSave : DtoForSaveKeyBase<int?>
    {

    }

    public class Tenant : TenantForSave
    {

    }

    public class TenantForClient : DtoBase
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Name2 { get; set; }
        public string ImageId { get; set; }
    }
}

File: BSharp/ClientApp/src/app/data/dto/tenant.ts
import { DtoForSaveKeyBase } from './dto-for-save-key-base';
import { DtoBase } from './dto-base';

export class TenantForSave extends DtoForSaveKeyBase {

}

export class Tenant extends TenantForSave {

}

export class UserCompany extends DtoBase {
    Id: number;
    Name: string;
    Name2: string;
    Name3: string;
    ImageId: string;
}
